test/consensus-gateway/data/erc20_deposit_proof.json

```diff
@@ -0,0 +1,40 @@
+            "0xf90151a0620752d6bea8de401a8c510032e39222b65b041db368434fcc14a225eb1b85cba0d824ad267b926051a297bdf654a1edbe5faa6d0df67a3dc0ab774b73fd07617c8080a0b1943f25819a440218bea1a76a008e0b58dd5159230588ce626c5bc46c14b636a084738c7684996d2818b5afb5c90a43b7c77fe79a7979f7e99bbd0f0b5088d7cea03cd7412ba749e646a075e36efa26b3c9027d6575c746a7c5858fd91f73fb9a008080a00b223f60e135cec31dbd69a809d9b93c5cd1ac17600d9be3301b35c339cd2505a0d5f5c04261faafc31381af77e7ccf55889b3234899271c1f6aae194bbb05dd98a03cca1eb0ee4bbf953527c1da5f3bc3f76caceb445acd5bd73a1177a19e2d76fb80a0ee404b58868dd330132e9015883deb72d80aefa4814784a69d7295b9b1c60cd580a08abe8247fce754b95926e9217ec2ee42030eb22c08a7808f452d386b14b055b180",
+            "0xf851808080808080808080a0c422d96bdc268aada9aac5f47ff648323754a5f0a7f4f70a1aa05bf88243c213808080a0ffb2430e5279a9c5e4925bb74902cfbb056087166f7fe6b80f3a07ae751c9b5f808080",
+            "0xf8689f34decd1c13ab57edcdb1c9964ce4c80e2b8da8f840c19ddbe411aa1e920854b846f8440180a097c1dad42f49a30c411675156778001421a8feedb9154aa5913330da95e78219a0b67d696ce3291dd9337a5d2ee50dbb223ba581f90426cf79c26e27684889187d"
+        ],
+        "balance": "0",
+        "codeHash": "0xb67d696ce3291dd9337a5d2ee50dbb223ba581f90426cf79c26e27684889187d",
+        "nonce": "1",
+        "storageHash": "0x97c1dad42f49a30c411675156778001421a8feedb9154aa5913330da95e78219",
+        "storageProof": [
+            {
+                "key": "0x3c0862f5cfa9641ffd13063c3426c19b6a36d4ad2e733f1781f32a470f29c1ab",
+                "value": "1",
+                "proof": [
+                    "0xf90131a0f1598eaa2023b6266475515e53c809c95e38daa4da7b020dcdbc838571f65c29a04eb5e55e49a260024c7b5a32bbabc8fc8e15348f82c4a2f56839ab298985ba3f8080a01027485f15ac1a4ac02cd20519bc4921bec01630ebebd7b59088ee43db83c95c8080a00ecd7e6d3e56fd71b27ca6f6487bb9060888b3b5f7ea062c4eef19831dbc8a16a034e54e311f93014fb93fe797389092db6fa05c8d12963dc0abdb9c717659049780a0330632230a9099cb5b61d6699e318e33324501259a9e51077a95f22720c4eb7e80a09c70929093a03ad9d0d1a7a825835508ca4933b4bc189269b755e0e8560a1230a0a6af0f19cbf24be0a6c1a5f3e5091fc5301c9a8590a13d4301b3056602182f7a80a078eda04c6e4ed5477ba5feb50f6ee96255aa68544e9411451889daca8df71ea880",
+                    "0xe2a03974e68814ff04408d6d120780a4a0ecde1d9aa33e1a9dc137486e8549a6b63a01"
+                ]
+            }
+        ]
+    }
+}
```

Comment:
```suggestion
}

```

Reply:
File deleted, due to the same reason as above.